src/openzaak/components/zaken/api/mixins.py

```diff
@@ -74,3 +75,17 @@ def perform_destroy(self, instance: models.Model) -> None:
         zaak = instance.zaak
         self._check_zaak_closed(zaak)
         super().perform_destroy(instance)
+
+
+class ExpandMixin:
```

Comment:
This seems more generic than zaken? Maybe move it to a higher level once documents API is up for expand :)

Reply:
The mixin is moved to utils